DXRando/DeusEx/Classes/DXRLamp2.uc

```diff
@@ -1,17 +1,60 @@
 //=============================================================================
-// Lamp2 except Warm White.  Fuck daylight bulbs.
+// Lamp2 except with a warm white light at the top.
+// Fuck daylight bulbs.
 // Localization files overwrite the ItemName,
 // so really force it to not be halogen
 //=============================================================================
 class DXRLamp2 injects #var(prefix)Lamp2;
 
+var DynamicLight lt;
+
+function InitLight()
+{
+    local vector loc;
+
+    if (lt == None) {
+        lt = Spawn(class'DynamicLight');
```

Comment:
you don't need to call SetLocation below if you just spawn it with the correct location to begin with
```suggestion
        lt = Spawn(class'DynamicLight',,, loc);
```